tex/chapters/Introduction.tex

```diff
@@ -1,50 +1,163 @@
+%!TEX root = ../main.tex
+
 \chapter{Introduction}\label{cha:introduction}
 
-\section{Introduction}
+The Prisoner's Dilemma (PD) is a classical model in Game Theory.
+It is a simple two player game where the agents must make a binary decision without communicating.
+This is often presented as two suspects who have been arrested and are being interrogated separately.
+They have the option of whether to cooperate with each other or defect and each player receives an indivual payoff that depends on the actions that have been taken.
 
-There is this data that is pretty awesome, I'm going to plot it and show it to
-you in sections~\ref{sec:the_data} and~\ref{sec:the_distribution_of_the_data}.
+Now, consider the situation that both prisoners return to their cells each night, to be presented with the same choice the next day.
+Furthermore, allow this process to continue repeatadely. 
+This is called the Iterated Prisoner's Dilemma (IPD) and has been an object of interest ever sincethe 1950's but became more popular after Robert Axelrod's large amount of work in the 1980's.
 
-\lipsum
+Several computer based IPD tournament have been held over the years, but the first was held by Axelrod in 1980 (use our paper for refs).
+Subsequently, two anniversary tournaments were held in 2004 and finally there was the Stewart and Plotkin 2012 tournament.
+The original source code of these is only available for Axelrod's second tournament (in FORTRAN) and it is not well documented, tested or easily re-useable.
 
-\section{The data}\label{sec:the_data}
+Recently, a group of scientists have been working to improve this situation by producing an open source version of Axelrod's original tournament ~\cite{Knight2016}.
+Referred to as the Axelrod-Python library, it aims to provide a resource for the design of new strategies and interactions between them, as well as conducting tournaments and ecologi-cal simulations for populations of strategies.
+At the time of writing, the library has 139 strategies implemented with the capability to run evolutionary tournaments and tournaments on different topologies.
```

Comment:
This is already not true anymore. I suggest you go for something like: "The version of the library used in this work will be vx.y.z \cite{} and contains N strategies." More strategies keep on getting added. 

At the same time, on gitter just now I've suggested you implement other things that will be very useful in further chapters so no doubt x.y.z will change.